pyproject.toml

```diff
@@ -8,20 +8,9 @@ authors = []
 python = "^3.7"
 flask = "^1.1.2"
 python-dotenv = "^0.14.0"
```

Comment:
You are missing required packages.

You should have requests, gunicorn, pytest, selenium, pytest-mock.

Ideally the ones just for testing should be in the dev-dependencies section.